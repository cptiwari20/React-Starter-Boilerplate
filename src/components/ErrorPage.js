import React from 'react';
import { Link } from 'react-router-dom';

const ErrorPage = () => (
  <div>
    <h1>Sorry! No page found.</h1>
    <p>Go to Dashboard.</p>
    <Link to='/'>Home</Link>
  </div>
);

export default ErrorPage;