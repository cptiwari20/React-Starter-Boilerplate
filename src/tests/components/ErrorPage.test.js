import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from '../../components/ErrorPage';

it('should correctly render Error Page', () => {
  const wrapper = shallow(<ErrorPage />)
  expect(wrapper).toMatchSnapshot();
})
