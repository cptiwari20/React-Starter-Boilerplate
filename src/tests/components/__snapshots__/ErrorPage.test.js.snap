// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Error Page 1`] = `
<div>
  <h1>
    Sorry! No page found.
  </h1>
  <p>
    Go to Dashboard.
  </p>
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
